docs/content/drafts/SelectPanel.mdx

```diff
@@ -0,0 +1,36 @@
+---
+componentId: selectpanel_v2
+title: SelectPanel v2
+status: Draft
+source: https://github.com/primer/react/blob/main/src/drafts/SelectPanel2
+storybook: '/react/storybook/?path=/story/drafts-components-selectpanel'
+---
+
+import data from '../../../src/drafts/SelectPanel2/SelectPanel.docs.json'
+
+A `SelectPanel` provides an anchor that will open an overlay with a list of selectable items, and a text input to filter the selectable items
```

Comment:
Is this a good place to describe more differences between this vs `ActionMenu` and that this component replaces `FilteredList` ?

Reply:
Not anymore :)

That job is now done by the design guidelines: https://primer.style/components/selectpanel

SelectPanel.mdx in this repo is temporary because primer/react docs wouldn't be around for much longer